iahx/templates/custom/base.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html dir="ltr" lang="{{lang}}" xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
Este atributo server para informar se o texto da página é de leitura left to right ou right to left (idioma árabe, etc).
http://www.w3schools.com/tags/att_global_dir.asp

Estarei removendo do código pois não esta previsto o uso de idiomas com leitura right to left.